plugins/nomad/src/Router.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Entity } from '@backstage/catalog-model';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { MissingAnnotationEmptyState } from '@backstage/core-components';
+import { Route, Routes } from 'react-router-dom';
+import { EntityNomadAllocationListTable } from './components/EntityNomadAllocationListTable/EntityNomadAllocationListTable';
+
+/** @public */
+export const NOMAD_NAMESPACE_ANNOTATION = 'nomad.io/namespace';
+
+/** @public */
+export const NOMAD_JOB_ID_ANNOTATION = 'nomad.io/job-id';
+
+/** @public */
+export const NOMAD_GROUP_ANNOTATION = 'nomad.io/group';
+
+/** @public */
+export const isNomadJobIDAvailable = (entity: Entity) =>
+  Boolean(entity.metadata.annotations?.[NOMAD_JOB_ID_ANNOTATION]);
+
+/** @public */
+export const isNomadAllocationsAvailable = (entity: Entity) =>
+  Boolean(entity.metadata.annotations?.[NOMAD_JOB_ID_ANNOTATION]) ||
+  Boolean(entity.metadata.annotations?.[NOMAD_GROUP_ANNOTATION]);
+
+/** @public */
+export const EmbeddedRouter = () => {
```

Comment:
does need to be exported? I'm thinking it would be enough with `EntityNomadContent` and leaving this internal.

Reply:
I _think_ this is what you mean but lmk if I have this wrong and you mean literally not `export`ing the `Router`. Apologies if this is only part of it: https://github.com/backstage/backstage/pull/18010/commits/031b8bed5aa50b5fdac0fddb5902cfb067b076fd

I've removed it from the API (no `@public`)

Reply:
Perfect! 👍